corehq/ex-submodules/pillowtop/es_utils.py

```diff
@@ -191,7 +187,12 @@ def initialize_mapping_if_necessary(es, index_info):
         pillow_logging.info("Initializing elasticsearch mapping for [%s]" % index_info.type)
         mapping = copy(index_info.mapping)
         mapping['_meta']['created'] = datetime.isoformat(datetime.utcnow())
-        mapping_res = es.indices.put_mapping(index_info.type, {index_info.type: mapping}, index=index_info.index)
+        mapping_res = es.indices.put_mapping(
+            index_info.type,
+            {index_info.type: mapping},
+            index=index_info.index,
+            include_type_name=True
```

Comment:
what does this do?

Reply:
@snopoke more infomation why we need include_type_name you can find https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-put-mapping.html